res/enkf/local_config.py

```diff
@@ -136,7 +124,7 @@ def getMinistep(self, mini_step_key):
     def getObsdata(self, obsdata_key):
```

Comment:
As far as I can tell this functionality is never called in the tests? Can you add a test to make sure the behavior stays the same?

Reply:
Tested here: #3046